internal/metrics/metrics.go

```diff
@@ -203,6 +209,31 @@ func (s *ScheduleTimer) Mark(ctx context.Context, role, outcome string) {
 	}
 }
 
+// ReportDriverCollocationMetric reports metrics around whether driver collocation with executors. This is meant to be used
```

Comment:
nit nit: ...whether driver is collocated...

Reply:
yup, shouldn't blindly trust goland refactor